pkgs/os-specific/linux/usbutils/default.nix

```diff
@@ -14,6 +14,7 @@ stdenv.mkDerivation rec {
   postInstall =
     ''
       substituteInPlace $out/bin/lsusb.py \
+        --replace "/usr/bin/env python3" ${python3}/bin/python3 \
         --replace /usr/share/usb.ids ${hwdata}/share/hwdata/usb.ids
```

Comment:
This is broken, try https://github.com/NixOS/nixpkgs/commit/c667c413e1c45d23c79bec26d60d38a0b1001b77